src/core/display/Container.js

```diff
@@ -386,6 +386,31 @@ export default class Container extends DisplayObject
         // FILL IN//
     }
 
+    /**
+     * Renders all the object based on renderChildrenFirst
+     *
+     * @private
+     * @param {PIXI.WebGLRenderer} renderer - The renderer
+     */
```

Comment:
Not sure about the name of the method. Maybe something like "_renderWebGLObjects"? Since it only deals with WebGL case.  Any ideas?

Reply:
You are right. This name is better